radicle-node/src/service.rs

```diff
@@ -595,11 +600,18 @@ where
                     Ok(updated) => {
                         log::debug!(target: "service", "Fetched {rid} from {remote}");
 
-                        self.reactor.event(Event::RefsFetched {
+                        let event = Event::RefsFetched {
                             remote,
                             rid,
                             updated: updated.clone(),
-                        });
+                        };
+                        self.reactor.event(event.clone());
+                        self.emitter
+                            .subscribers
+                            .lock()
+                            .unwrap()
+                            .retain(|s| s.try_send(event.clone()).is_ok());
```

Comment:
I didn't change the emit type yet, though.

Reply:
Yeah this should be a method on `Emitter` called `publish`

Reply:
went with [`emit`](https://github.com/radicle-dev/heartwood/compare/151868ed3c0ec50054de690b24126c57aa5e247a..4a65b33f3ac878437c41bafce9fa3993e0e357ce) instead since it's what [nakamoto](https://github.com/cloudhead/nakamoto/blob/master/net/src/event.rs#L43) uses as well.